src/js/tech/ChromecastTech.js

```diff
@@ -364,6 +366,30 @@ ChromecastTech = {
       return undefined;
    },
 
+   /**
+    * This function is "required" when implementing {@link external:Tech} and is supposed
+    * to return a mock
+    * {@link https://developer.mozilla.org/en-US/docs/Web/API/TimeRanges|TimeRanges}
+    * object that represents the portions of the current media item that has playable
+    * content. However, the Chromecast API does not currently provide a way to determine
+    * how much the media item has playable content, so we'll just assume the entire video
+    * is an available seek target.
+    *
+    * The risk here lies in DVR, where ther may exist a sliding window and seeking is only
```

Comment:
For readers that are not familiar with the `DVR` acronym, could we either:

1) Link to a resource (preferably some official documentation somewhere) that explains what `DVR` is, or:
2) Use a more familiar, generic term like `live streaming`

Reply:
Small typo: `where ther` -> `where there`.

Also, it may be slightly more clear to say `sliding window of playable content` rather than just `sliding window`.